src/database/migrations/20200224160514-update-notifications-table.js

```diff
@@ -0,0 +1,40 @@
+
+export default {
+  up(queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.addColumn(
+        'Notifications',
+        'userID',
+        {
+          allowNull: true,
+          type: Sequelize.STRING
+        },
+      ),
+      queryInterface.addColumn(
+        'Notifications',
+        'names',
+        {
+          allowNull: true,
+          type: Sequelize.STRING
+        },
+      ),
+      queryInterface.addColumn(
+        'Notifications',
+        'imageUrl',
+        {
+          allowNull: true,
+          type: Sequelize.STRING
+        },
+      ),
+    ])
+  },
+
+  down(queryInterface) {
+    return Promise.all([
+      queryInterface.removeColumn('Notifications', 'userID'),
+      queryInterface.removeColumn('Notifications', 'names'),
```

Comment:
@RusimbiP why are you adding these columns to the notifications table?

Reply:
I need them to save that data

Reply:
@RusimbiP again you can make use of relationships in place instead of duplicating those.

Reply:
This increases the workload on the front end. Copying data from the other table is not always bad.

Reply:
Fixed